requirements.txt

```diff
@@ -24,3 +24,11 @@ PyYAML
 simplemediawiki
 sqlalchemy
 waitress
+flake8
+pytest
+pytest-cov
+webtest
+conu >= 0.5.0
+munch
+psycopg2
+requests
```

Comment:
This change shouldn't be part of this PR (`git commit -a` considered harmful? :wink:), and these modules don't belong in `requirements.txt` anyway, they're only needed for testing, not running Bodhi.

Reply:
My bad! Got too used to committing all...